torchreid/engine/builder.py

```diff
@@ -4,14 +4,15 @@
 
 from .image import (ImageAMSoftmaxEngine, MultilabelEngine, MultiheadEngine)
 
+NNCF_ENABLED_LOSS = ['softmax', 'am_softmax', 'am_binary']
```

Comment:
For multihead classification, loss_name comes as `softmax,asl` , so asl will raise error I think. Could you revise this line to include `asl`?